src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -180,6 +178,11 @@ private IFilteredTemplateInfo UseSecondaryCriteriaToDisambiguateTemplateMatches(
 
         public static int Run(string commandName, ITemplateEngineHost host, Action<IEngineEnvironmentSettings, IInstaller> onFirstRun, string[] args)
         {
+            if (args.Any(x => string.Equals(x, "--debug:attach", StringComparison.Ordinal)))
```

Comment:
Nice way to get around needing to parse the args to get at this flag. Makes it easier to attach earlier 👍